File: worker/worker-impl/src/main/java/com/masterello/worker/client/dto/CategoryDTO.java
package com.masterello.worker.client.dto;

import java.time.OffsetDateTime;
import java.util.UUID;

public record CategoryDTO(UUID uuid, String name, String description, int categoryCode, Integer parentCode,
                   boolean isService, OffsetDateTime createdDate, OffsetDateTime updatedDate, boolean active) {
}

File: category/category-contract/src/main/kotlin/com/masterello/category/dto/CategoryDto.kt
package com.masterello.category.dto

import java.time.OffsetDateTime
import java.util.*

data class CategoryDto (
        val uuid: UUID? = null,
        val name: String? = null,
        val description: String? = null,
        val categoryCode: Int? = null,
        val parentCode: Int? = null,
        val isService: Boolean? = null,
        val createdDate: OffsetDateTime? = null,
        val updatedDate: OffsetDateTime? = null,
        val active: Boolean? = null
)
